### PR TITLE
Put credential in the cookies

### PR DESCRIPTION
**Pull request by fgervasi**
[Issue#27](https://github.com/mitra-social/mitra-frontend/issues/27)

### CODE REVIEW

File: src/utils/authentication-util.ts [hunk 1]
```diff
@@ -1,27 +1,41 @@
...
+
 export class AuthenticationUtil {
   private static readonly USER = "user";
   private static readonly USER_TOKEN = "user-token";
 
+  // Attributes for cookies
+  private static readonly ATTRS = {
+    expires: 1,
+    secure: true,
+    samesite: "lax",
```
Comment (TiMESPLiNTER) [inline src/utils/authentication-util.ts:11]: there's a httpOnly but I guess that's not useful as it doesn't get sent with XHR requests then, right?

File: src/utils/authentication-util.ts [hunk 2]
```diff
@@ -1,27 +1,41 @@
+import Cookies from "js-cookie";
+
 export class AuthenticationUtil {
   private static readonly USER = "user";
   private static readonly USER_TOKEN = "user-token";
 
+  // Attributes for cookies
+  private static readonly ATTRS = {
+    expires: 1,
```
Comment (TiMESPLiNTER) [inline src/utils/authentication-util.ts:9]: why 1? Does it mean "never"?

File: src/utils/authentication-util.ts (hunk 1 above)
Comment (fgervasi) [reply src/utils/authentication-util.ts:11]: We do not have any XHR requests, so we do not need this attribute

File: src/utils/authentication-util.ts (hunk 2 above)
Comment (fgervasi) [reply src/utils/authentication-util.ts:9]: 1 means days. the js-cookie library you can enter the number of days or set a date when the cookie has expired
[js-cookie#expires](https://github.com/js-cookie/js-cookie#expires)

File: src/utils/authentication-util.ts (hunk 1 above)
Comment (TiMESPLiNTER) [reply src/utils/authentication-util.ts:11]: We don't? Then we should set the httpOnly to true.

File: src/utils/authentication-util.ts (hunk 2 above)
Comment (fgervasi) [reply src/utils/authentication-util.ts:9]: Perhaps a constant that describes what this magic number is would be good

Comment (TiMESPLiNTER) [reply src/utils/authentication-util.ts:9]: very good idea :D

File: src/utils/authentication-util.ts [hunk 3]
```diff
@@ -3,14 +3,24 @@ import Cookies from "js-cookie";
 export class AuthenticationUtil {
   private static readonly USER = "user";
   private static readonly USER_TOKEN = "user-token";
+  private static readonly EXPIRED_IN_DAYS = 1;
```
Comment (TiMESPLiNTER) [inline src/utils/authentication-util.ts:6]:
```suggestion
  private static readonly EXPIRATION_IN_DAYS = 1;
```

File: src/utils/authentication-util.ts [hunk 4]
```diff
@@ -3,14 +3,24 @@ import Cookies from "js-cookie";
...
 
   // Attributes for cookies
-  private static readonly ATTRS = {
-    expires: 1,
+  private static readonly ATTRS_FOR_PRODUCTION = {
+    expires: AuthenticationUtil.EXPIRED_IN_DAYS,
```
Comment (TiMESPLiNTER) [inline src/utils/authentication-util.ts:10]:
```suggestion
    expires: AuthenticationUtil.EXPIRATION_IN_DAYS,
```

**Pull request merged**